src/redux/reducers/authReducer.js

```diff
@@ -0,0 +1,44 @@
+import {
+  SIGNUP_ERROR,
+  SIGNUP_SUCCESS,
+  START_FETCH,
+  SERVER_ERROR,
+  CLEAR_ERROR
+} from "../action_types";
+import initialState from "./initialState";
```

Comment:
You've imported `initialState` from `initialState`. But you haven't specified what exactly you require from this `initialState`. Remember everyone will be making use of this same `initialState`. So wouldn't it be better if you kept contents for this specific reducer separate from everyone elses content.

In this case, refactor `initialState` to:

const initialState = {
  authUser: {
      message: "",
      error: {},
      fetching: false,
      fetched: false,
      success: false
},
...
};

When someone else needs to make use of the initialState, they'll create their own object right after yours, where I've placed the ellipses.
In this case, you can now access `initialState.authUser`

Reply:
On it as well

Reply:
Refactored initialState appropriately